.github/workflows/accSelector.yaml

```diff
@@ -0,0 +1,46 @@
+name: SetM365AppAcc
```

Comment:
This file looks identical to one in https://github.com/alcionai/corso/pull/3288

Should we just review this PR? :)

Reply:
For CI test to run and pass we will need the `accSelector.yaml` to be already present in main because of 

https://github.com/alcionai/corso/blob/f1d4da141cfe9ed1e129abbb0b7c43b2ab16ea7d/.github/workflows/ci.yml#L33

which requires the file to be present in main. That is why I wanted to merge https://github.com/alcionai/corso/pull/3288 first and then this PR can go in.